lib/wizard/step_updater.rb

```diff
@@ -37,6 +37,10 @@ def update_setting(id, value)
         value = Upload.get_from_url(value) || ''
       end
 
+      if id == :navigation_menu
+        value = value.to_s == "true" ? NavigationMenuSiteSetting::SIDEBAR : NavigationMenuSiteSetting::LEGACY
```

Comment:
```suggestion
        value = value.to_s == "true" ? NavigationMenuSiteSetting::SIDEBAR : NavigationMenuSiteSetting::HEADER_DROPDOWN
```

@mcwumbly Is the goal to still drop back to legacy here?

Reply:
@tgxworld no, I think you have it right. The toggle here should be choosing between `SIDEBAR` and `HEADER_DROPDOWN`.  We can reserve `LEGACY` for existing sites (new sites can go out of their way to choose it).

Reply:
@oblakeerickson looks like a test also needs updating?

```
  1) Wizard::StepUpdater privacy updates to open correctly
     Failure/Error: expect(SiteSetting.navigation_menu).to eq(NavigationMenuSiteSetting::LEGACY)

       expected: "legacy"
            got: "header dropdown"
     ```